src/main/java/fi/metatavu/metaform/server/metaforms/FieldController.java

```diff
@@ -136,16 +149,38 @@ public Object getFieldValue(Metaform metaformEntity, Reply reply, String fieldNa
         .collect(Collectors.toList());
     } else if (field instanceof AttachmentReplyField) {
       return attachmentReplyFieldItemDAO.listByField((AttachmentReplyField) field).stream()
-          .map(AttachmentReplyFieldItem::getAttachment)
-          .map(Attachment::getId)
-          .map(UUID::toString)
-          .collect(Collectors.toList());      
+        .map(AttachmentReplyFieldItem::getAttachment)
+        .map(Attachment::getId)
+        .map(UUID::toString)
+        .collect(Collectors.toList());      
+    } else if (field instanceof TableReplyField) {
+      return tableReplyFieldRowDAO.listByField((TableReplyField) field).stream()
+        .map(this::getTableRowValue).collect(Collectors.toList());
     } else {
       logger.error("Could not resolve {}", fieldName); 
     }
     
     return null;
   }
+
+  /**
+   * Returns table field row as map
+   * 
+   * @param row row
+   * @return table field row as map
+   */
+  private Map<String, Object> getTableRowValue(TableReplyFieldRow row) {
+    return anyTableReplyFieldRowCellDAO.listByRow(row).stream()
+      .collect(Collectors.toMap(TableReplyFieldRowCell::getName, (cell) -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the parentheses around the "cell" parameter ([squid:S1611](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1611))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=60&issues=AWY4b1ppvOWp7Yd6rqji&open=AWY4b1ppvOWp7Yd6rqji)

[ISSUE_KEY]: AWY4b1ppvOWp7Yd6rqji